### PR TITLE
[RFR] Field wizard

### PR DESCRIPTION
**Pull request by djhi**

### CODE REVIEW

File: src/app/js/admin/fields/wizard/index.js [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+    modal: {
+        maxWidth: '100%',
+    },
+    column: {
+        minWidth: '10rem',
+        maxWidth: '10rem',
+    },
+};
+
+const getActions = memoize((
```
Comment (ThieryMichel) [inline src/app/js/admin/fields/wizard/index.js]: make this a component in a separate file

File: src/app/js/admin/fields/wizard/StepDisplay.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import PositionField from '../PositionField';
+import Format from '../../FormatEdition';
+import { polyglot as polyglotPropTypes, field as fieldPropTypes } from '../../../propTypes';
+
+export const StepDisplayComponent = ({
+    field,
+    p: polyglot,
+    ...props
+}) => (
+    <Step label="field_wizard_step_display" {...props}>
```
Comment (ThieryMichel) [inline src/app/js/admin/fields/wizard/StepDisplay.js:17]: How do we tell which props come from here and which props come from reduxForm. And how to be sure which props is actually needed?
List all props that need to be passed, ignore the rest.

File: src/app/js/admin/fields/wizard/StepIdentity.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        <MenuItem
+            className={`completes_${f.label.toLowerCase().replace(/\s/g, '_')}`}
+            key={f.name}
+            value={f.name}
+            primaryText={f.label}
+        />
+    ));
+
+    return (
+        <Step label="field_wizard_step_identity" {...props}>
```
Comment (ThieryMichel) [inline src/app/js/admin/fields/wizard/StepIdentity.js]: same

File: src/app/js/admin/fields/wizard/StepSearch.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import compose from 'recompose/compose';
+import translate from 'redux-polyglot/translate';
+import { Field } from 'redux-form';
+
+import Step from './Step';
+import FormCheckboxField from '../../../lib/FormCheckboxField';
+import { polyglot as polyglotPropTypes } from '../../../propTypes';
+
+export const StepSearchComponent = ({ p: polyglot, ...props }) => (
```
Comment (ThieryMichel) [inline src/app/js/admin/fields/wizard/StepSearch.js:10]: same

File: src/app/js/admin/fields/wizard/StepSemantics.js [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+        <MenuItem
+            className={`completes_${f.label.toLowerCase().replace(/\s/g, '_')}`}
+            key={f.name}
+            value={f.name}
+            primaryText={f.label}
+        />
+    ));
+
+    return (
+        <Step label="field_wizard_step_semantic" {...props}>
```
Comment (ThieryMichel) [inline src/app/js/admin/fields/wizard/StepSemantics.js:28]: same

File: src/app/js/admin/fields/wizard/StepValue.js [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+import { getTransformerMetas } from '../../../../../common/transformers';
+
+import StepValueValue from './StepValueValue';
+import StepValueColumn from './StepValueColumn';
+
+export const StepValueComponent = ({
+    datasetFields,
+    field,
+    handleChange,
+    ...props
```
Comment (ThieryMichel) [inline src/app/js/admin/fields/wizard/StepValue.js:19]: same

File: src/app/js/admin/fields/wizard/StepSemanticsComposition.js [hunk 7]
```diff
@@ -0,0 +1,173 @@
...
+    handleRemoveColumn,
+    p: polyglot,
+    separator,
+}) => (
+    <div>
+        <Subheader style={styles.header}>{polyglot.t('composed_of')}</Subheader>
+        <div style={styles.inset}>
+            {columns.map((col, index) => (
+                <div
+                    key={`compotransformerssition_${index}`} // eslint-disable-line
```
Comment (ThieryMichel) [inline src/app/js/admin/fields/wizard/StepSemanticsComposition.js]: compotransformerssition ?

**Review by ThieryMichel — CHANGES_REQUESTED** [review]

File: src/app/js/admin/fields/wizard/index.js (hunk 1 above)
Comment (djhi) [reply src/app/js/admin/fields/wizard/index.js]: Can't do, I have to return an array

**Review by ThieryMichel — APPROVED** [review]

**Pull request merged**